Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -4115,6 +4115,156 @@ script:
+      name: expiration_date
+      required: false
+      isArray: false
+      type: String
+    - description: Comment string.
+      name: comment
+      required: false
+      isArray: false
+      type: String
+    - auto: PREDEFINED
+      description: Keyword representing the indicator's reputation.
+      name: reputation
+      predefined:
+        - 'GOOD'
+        - 'BAD'
+        - 'SUSPICIOUS'
+        - 'UNKNOWN'
+      required: false
+      isArray: false
+      type: String
+    - auto: PREDEFINED
+      description: Character representing the indicator's reliability rating. Valid values are A-F. A is the most reliable, F is the least.
+      name: reliability
+      predefined:
+        - 'A'
+        - 'B'
+        - 'C'
+        - 'D'
+        - 'E'
+        - 'F'
+      required: false
+      isArray: false
+      type: String
+    - description: String representing the indicator class (for example, "Malware").
+      name: class
+      required: false
+      isArray: false
+      type: String
+    - description: String representing the name of the vendor who reported this indicator.
+      name: vendor_name
+      required: false
+      isArray: false
+      type: String
+    - auto: PREDEFINED
+      description: Keyword representing the vendor's reputation. Required when vendor_name is defined.
+      name: vendor_reputation
+      predefined:
+        - 'GOOD'
+        - 'BAD'
+        - 'SUSPICIOUS'
+        - 'UNKNOWN'
+      required: false
+      isArray: false
+      type: String
+    - auto: PREDEFINED
+      description: Character representing the vendor's reliability rating. Valid values are A-F. A is the most reliable, F is the least. Required when vendor_reputation is defined.
+      name: vendor_reliability
+      predefined:
+        - 'A'
+        - 'B'
+        - 'C'
+        - 'D'
+        - 'E'
+        - 'F'
+      required: false
+      isArray: false
+      type: String
+    - auto: PREDEFINED
+      description: The input format selected, which determines how parameters are sent to the XDR API.
+      defaultValue: 'JSON'
+      name: input_format
+      predefined:
+        - 'CSV'
+        - 'JSON'
+      required: false
+      isArray: false
+      type: String
+    - description: "The JSON/CSV object contains the IOC details. Make sure a valid IOC object is passed."
+      name: ioc_object
+      required: false
+      isArray: false
+      type: String
+    description: "Upload IOCs rules to XSIAM. When `ioc_object` is defined, disregard any other provided arguments as `ioc_object` takes precedence.
+                  If `vendor_name`, `vendor_reputation`, and `vendor_reliability` are used, only a single vendor is supported. For multiple vendors, utilize an `ioc_object` in JSON format.
+                      Adding rule with the same indicator and different parameters will result in an update to the existing rule."
```

Comment:
Adding a rule with the same indicator, but with different parameters, will update the existing rule."